WebDriverAgentLib/Utilities/FBConfiguration.m

```diff
@@ -346,4 +345,27 @@ + (NSRange)bindingPortRangeFromArguments
   return NSMakeRange(port, 1);
 }
 
++ (void)setReduceMotionEnabled:(BOOL)isEnabled
+{
+  Class settingsClass = NSClassFromString(@"AXSettings");
+  AXSettings *settings = [settingsClass sharedInstance];
```

Comment:
is `settingsClass` always not nil?

Reply:
Yes, I think so. It exists over iOS7, at least
http://developer.limneos.net/index.php?ios=7.0&framework=AccessibilityUtilities.framework&header=AXSettings.h